### PR TITLE
CAM-12451: add HTML license books

### PR DESCRIPTION
**Pull request by ThorbenLindhauer**
- for platform and bridge

related to CAM-12451

### CODE REVIEW

File: content/introduction/licenses.md [hunk 1]
```diff
@@ -25,9 +25,13 @@ If you are using the Enterprise Platform, the software is provided under a licen
 
 ## Third-Party Libraries
 
-The Camunda BPM platform (Community and Enterprise Edition) includes third-party libraries developed by third parties (please see the documentation section [Third-Party Libraries][third-party-libraries])). For a complete list of license notices for libraries contained in the Camunda BPM platform, please refer to the [license book][license-book].
+The Camunda BPM platform (Community and Enterprise Edition) includes third-party libraries developed by third parties (please see the documentation section [Third-Party Libraries][third-party-libraries])). See the following resources:
```
Comment (tasso94) [inline content/introduction/licenses.md:28]:
```suggestion
The Camunda BPM platform (Community and Enterprise Edition) includes third-party libraries developed by third parties (please see the documentation section [Third-Party Libraries][third-party-libraries]). See the following resources:
```

File: content/introduction/licenses.md [hunk 2]
```diff
@@ -25,9 +25,13 @@ If you are using the Enterprise Platform, the software is provided under a licen
...
 
-[third-party-libraries]: {{< ref "/introduction/third-party-libraries.md" >}}
+* [HTML License Book Camunda BPM Platform]({{< ref "/introduction/third-party-libraries/camunda-bpm-platform-license-book.md" >}})
+* [Plain text license book Camunda BPM Platform][license-book]
+* [HTML License Book Camunda BPM RPA Bridge]({{< ref "/introduction/third-party-libraries/camunda-bpm-rpa-bridge-license-book.md" >}})
+
+[third-party-libraries]: {{< ref "/introduction/third-party-libraries/_index.md" >}}
 [license-book]: https://app.camunda.com/nexus/content/repositories/camunda-bpm/org/camunda/bpm/license-book/7.13.0/license-book-7.13.0.txt
```
Comment (tasso94) [inline content/introduction/licenses.md:35]:
```suggestion
[license-book]: https://app.camunda.com/nexus/content/repositories/camunda-bpm/org/camunda/bpm/license-book/7.14.0/license-book-7.14.0.txt
```

**Review by tasso94 — APPROVED** [review]
Looks good to me. Please consider the review hints below before merge:

File: content/introduction/licenses.md (hunk 2 above)
Comment (ThorbenLindhauer) [reply content/introduction/licenses.md:35]: Good catch, I somehow assumed the version number was generated :D

**Pull request merged**